bundle/metadata/annotations.yaml

```diff
@@ -5,7 +5,7 @@ annotations:
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: splunk-operator
   operators.operatorframework.io.bundle.channels.v1: alpha
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.18.1
+  operators.operatorframework.io.metrics.builder: operator-sdk-v1.15.0
```

Comment:
is this correct?

Reply:
Automerged. I will change the value.

Reply:
this will be regenerated with different PR to the develop once all the develop changes are done from 2.1.0